repose-aggregator/tests/performance-tests/roles/cloud_resource_create/tasks/create_server_gatling.yml

```diff
@@ -6,7 +6,11 @@
     flavor: performance1-4
     image: "{{ cloud.server.image }}"
     files: "{{ cloud.server.files }}"
-    region: "{{ cloud.region }}"
+    region: IAD
+    networks:
+      - public
+      - private
```

Comment:
Seemingly. I took `public` and `private` from the docs here:

>>> The network to attach to the instances. If specified, you must include ALL networks including the public and private interfaces. Can be id or label.

http://docs.ansible.com/ansible/rax_module.html